[2024-08-01T14:32:21.250+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-01T14:32:21.264+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_taxi_etl.upload_to_gcs scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2024-08-01T14:32:21.274+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_taxi_etl.upload_to_gcs scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2024-08-01T14:32:21.274+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-01T14:32:21.289+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_gcs> on 2023-01-02 00:00:00+00:00
[2024-08-01T14:32:21.300+0000] {standard_task_runner.py:57} INFO - Started process 287 to run task
[2024-08-01T14:32:21.305+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'nyc_taxi_etl', 'upload_to_gcs', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/nyc_taxi_etl.py', '--cfg-path', '/tmp/tmpycdsm21o']
[2024-08-01T14:32:21.307+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask upload_to_gcs
[2024-08-01T14:32:21.324+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-01T14:32:21.378+0000] {task_command.py:410} INFO - Running <TaskInstance: nyc_taxi_etl.upload_to_gcs scheduled__2023-01-02T00:00:00+00:00 [running]> on host 31f8257a797f
[2024-08-01T14:32:21.395+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-01T14:32:22.003+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nyc_taxi_etl' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2023-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-02T00:00:00+00:00'
[2024-08-01T14:32:22.004+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: upload_to_gcs() missing 1 required positional argument: 'credentials'
[2024-08-01T14:32:22.013+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=nyc_taxi_etl, task_id=upload_to_gcs, execution_date=20230102T000000, start_date=20240801T143221, end_date=20240801T143222
[2024-08-01T14:32:22.027+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 8 for task upload_to_gcs (upload_to_gcs() missing 1 required positional argument: 'credentials'; 287)
[2024-08-01T14:32:22.081+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2024-08-01T14:32:22.140+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-01T14:40:26.570+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-01T14:40:26.626+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_taxi_etl.upload_to_gcs scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2024-08-01T14:40:26.661+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_taxi_etl.upload_to_gcs scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2024-08-01T14:40:26.663+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-01T14:40:26.708+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_gcs> on 2023-01-02 00:00:00+00:00
[2024-08-01T14:40:26.759+0000] {standard_task_runner.py:57} INFO - Started process 783 to run task
[2024-08-01T14:40:26.770+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'nyc_taxi_etl', 'upload_to_gcs', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/nyc_taxi_etl.py', '--cfg-path', '/tmp/tmp301w2xse']
[2024-08-01T14:40:26.776+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask upload_to_gcs
[2024-08-01T14:40:26.835+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-01T14:40:26.906+0000] {task_command.py:410} INFO - Running <TaskInstance: nyc_taxi_etl.upload_to_gcs scheduled__2023-01-02T00:00:00+00:00 [running]> on host 31f8257a797f
[2024-08-01T14:40:26.942+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-01T14:40:27.552+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nyc_taxi_etl' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2023-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-02T00:00:00+00:00'
[2024-08-01T14:41:09.401+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-01T14:41:09.470+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=nyc_taxi_etl, task_id=upload_to_gcs, execution_date=20230102T000000, start_date=20240801T144026, end_date=20240801T144109
[2024-08-01T14:41:09.543+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
